datahub-web-react/src/app/entity/shared/containers/profile/EntityProfile.tsx

```diff
@@ -270,7 +270,7 @@ export const EntityProfile = <T, U>({
                                 headerActionItems={headerActionItems}
                                 subHeader={subHeader}
                             />
-                            <Divider />
+                            <Divider style={{ margin: '0' }} />
```

Comment:
I believe you want just `marginBottom: 0` since we still want the margin on the top otherwise it looks smooshed there